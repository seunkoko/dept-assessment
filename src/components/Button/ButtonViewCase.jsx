// eslint-disable-next-line
import React from 'react';

import './Button.scss';

const ButtonViewCase = (props) => {
	return (
    <button
      type="button"
      className="btn button-style view-case-style"
    >
      VIEW CASE
    </button>
	);
};

export default ButtonViewCase;
