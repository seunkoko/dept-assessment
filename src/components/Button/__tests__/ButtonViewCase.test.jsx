import React from 'react';

import { shallow } from 'enzyme';

import ButtonViewCase from '../ButtonViewCase';

describe('ButtonViewCase', () => {
	const component = shallow(<ButtonViewCase />);

	it('should render ButtonViewCase component', () => {
		expect(component).toMatchSnapshot();
  });
  
});