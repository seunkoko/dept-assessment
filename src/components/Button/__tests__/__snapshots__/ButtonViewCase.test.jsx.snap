// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonViewCase should render ButtonViewCase component 1`] = `
<button
  className="btn button-style view-case-style"
  type="button"
>
  VIEW CASE
</button>
`;
